docs/quickstart.rst

```diff
@@ -136,11 +136,18 @@ Now, suppose that you want to search the table for users with a last name
     for user in UserModel.query('Smith', UserModel.first_name.startswith('J')):
         print(user.first_name)
 
-You can combine query terms:
+You can combine query terms in filter conditions using OR:
```

Comment:
I would clarify by saying
> You can combine query terms in filter conditions using logical operators (ex, `|`, `&`, `==`, etc.)

Reply:
Thanks for the review.
I have given a separate example for `&` just below this one. Not sure how `==` would work here for combining queries 🤔 
Let me know if there are other logical operators that are supported here. I'll either add them in-line or add a separate example if they make more sense that way.